### ISSUE TITLE
Please consider updating the doc

### ISSUE THREAD

**Issue by toyssamurai** [YevhenKap/couchdb_dart#17]
It took me almost 30 mins to find out how to specify the host. The doc gives the following sample codes:

`final c = CouchDbClient(username: 'name', password: 'pass');`

Throughout the entire documentation, there's no mentioning of setting the host address. Even the example shows nothing about setting the host.

Yes, it's in the API doc of the CouchDbClient class and the test codes, but not having it in the main doc is very confusing.

**Comment by bean5** [comment]
@YevhenKap I agree that updating the documentation could be helpful in relation to this.
